src/main/java/pl/javastart/task/Main.java

```diff
@@ -1,8 +1,24 @@
 package pl.javastart.task;
 
+import java.util.Scanner;
+
 public class Main {
 
     public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+
+        for (int i = 0; i >= 0; i++) {
```

Comment:
To pętla nieskończona. Lepiej zapisać po prostu `while(true)` - wtedy na pierwszy rzut oka widać o co chodzi